src/main/java/de/tum/in/www1/exerciseapp/domain/SubmittedAnswer.java

```diff
@@ -19,6 +22,11 @@
 )
 @DiscriminatorValue(value="S")
 @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
```

Comment:
Please document the following lines properly so that other developers know why you are using these annotations